ui/src/Pages/Preferences/Invites/ManageInvites.jsx

```diff
@@ -34,18 +34,32 @@ function ManageInvites() {
     [dispatch]
   );
 
+  const toClipboard = (token) => {
+    navigator.clipboard.writeText(token).then(() => {
```

Comment:
While we handle the success of writing the token to the clipboard, we dont handle an error. We should add a callback that is called on error and creates a notification saying that we failed to copy to clipboard and shows the error